catalog/ui/search-ui/standard/server-impl.js

```diff
@@ -10,6 +10,8 @@
  *
  **/
 var URL = require('url'),
+    httpProxy = require('http-proxy'),
```

Comment:
let's revert this here because we don't actually need http-proxy to do what you want to do.